Makefile

```
@@ -0,0 +1,130 @@
+.PHONY: clean
+clean: ## Clean build artifacts
+	@echo "Cleaning build artifacts..."
+	@rm -rf coverage.txt
+	@rm -rf license-header-checker*
+	@go clean -cache -testcache
+
+.PHONY: verify
+verify: clean fmt test ## Verify code quality (fmt + test)
+
+.PHONY: lint
+lint: ## Run golangci-lint
```

checkmake (0.2.2)

[warning] Target body for "lint" exceeds allowed length of 5 (8).

(maxbodylength)